routes/keyword.js

```diff
@@ -36,10 +39,17 @@ router.get('/:word', (req, res) => {
   const word = req.params.word
   Keyword.find({ word })
     .lean()
-    .then(words => res.send(words[0].votes))
+    .then((wordList) => {
+      if (wordList.length === 0) {
```

Comment:
That's not reuse 😞. By using `.find` your causing the database to look at ALL the documents, whereas `findOne` would stop after the word is found. You can always do `res.send([word])` to send an array